packages/import/test/ValidateCsv.spec.ts

```diff
@@ -0,0 +1,307 @@
+            expect(items[1].tokenId).toBe("0x900edc98db53508e6742723988b872dd08cd09c2");
+        } else {
+            fail("items expected in result");
+        }
+    });
+
+    it("detects wrong CSV without header", async () => {
+        const result = await validateCsv.readItemsCsv(Readable.from(CSV_WITHOUT_FILE_WRONG_NO_HEADER));
+        if("error" in result) {
+            expect(result.error).toBe("Unexpected schema, check number of column and/or headers");
+        } else {
+            fail();
+        }
+    });
+
+    it("detects wrong CSV with bad headers", async () => {
+        const result = await validateCsv.readItemsCsv(Readable.from(CSV_WITHOUT_FILE_WRONG_BAD_HEADER));
+        if("error" in result) {
+            expect(result.error).toBe("Unexpected schema, check number of column and/or headers");
+        } else {
+            fail();
+        }
+    });
+
+    it("reads items with file with wrong type", async () => {
+        const result = await validateCsv.readItemsCsv(Readable.from(CSV_WITH_FILE_WRONG_TYPE));
+        if("items" in result) {
+            expect(result.fullyValidated).toEqual(false);
+            expect(result.rowType).toBe(CsvRowType.WithFile);
+            expect(result.errorSummary).toEqual({
+                "Invalid file content type": 2,
+            });
+
+            const items = result.items as CsvItemWithFile[];
+            expect(items.length).toBe(2);
+
+            expect(items[0].displayId).toBe("programming_music.jpg");
+            expect(items[0].id).toEqual(Hash.of("programming_music.jpg"));
+            expect(items[0].description).toBe("Programming Music");
+            expect(items[0].fileName).toBe("programming_music.jpg");
+            expect(items[0].fileContentType).toBe("wrong/type");
+            expect(items[0].fileSize).toBe("90718");
+            expect(items[0].fileHash).toEqual(Hash.fromHex("0xa025ca5f086f3b6df1ca96c235c4daff57083bbd4c9320a3013e787849f9fffa"));
+            expect(items[0].termsAndConditionsType).toBe("CC4.0");
+            expect(items[0].termsAndConditionsParameters).toBe("BY");
+
+            expect(items[1].displayId).toBe("lucas_games_characters.jpg");
+            expect(items[1].id).toEqual(Hash.of("lucas_games_characters.jpg"));
+            expect(items[1].description).toBe("LucasArts Games Characters");
+            expect(items[1].fileName).toBe("lucas_games_characters.jpg");
+            expect(items[1].fileContentType).toBe("wrong/type");
+            expect(items[1].fileSize).toBe("91880");
+            expect(items[1].fileHash).toEqual(Hash.fromHex("0x546b3a31d340681f4c80d84ab317bbd85870e340d3c2feb24d0aceddf6f2fd31"));
+            expect(items[1].termsAndConditionsType).toBe("CC4.0");
+            expect(items[1].termsAndConditionsParameters).toBe("BY-NC");
+        } else {
+            fail("items expected in result");
+        }
+    });
+
+    fit("reads items with token with wrong token", async () => {
```

Comment:
Forgotten `fit`.

Reply:
I have force-pushed the forgotten `fit`and `fdescribe`.